### PR TITLE
Fixed progress bar in Boards page

### PR DESCRIPTION
**Pull request by muskan2622**
### Functionality:
On creating new sprint progress bar was not working.

### Solution:
Now On completing the sprint on the same day its working

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
1. Go to boards page.
2. Create New sprint and mark it as completed.

### CODE REVIEW

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 1]
```diff
@@ -63,8 +64,10 @@ export class SprintDetailsComponent implements OnInit {
     const callable = this.functions.httpsCallable('sprints/updateSprintStatus');
     const appKey = this.backendService.getOrganizationAppKey();
     const uid = this.authService.getLoggedInUser();
+    console.log(this.today);
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.ts:67]: remove console.log

File: src/app/body/view-organization-details/teams-card/teams-card.component.html [hunk 2]
```diff
@@ -15,11 +15,11 @@
     <div class="card-body pt-4">
         <ng-container>
             <div class="row">
-                <div class="col-auto float-left">
+                <div class="col-md-10 float-left">
                     <h2 class="card-title my-0" (click)="openTeamDetails()" style="cursor: pointer;">{{ team.TeamName }}</h2>
                     <h3 class="card-subtitle text-muted">{{ team.TeamDescription }}</h3>
                 </div>
-                    <span class="material-icons float-left ml-0 mt-2" type="button" (click)="openTeamDetails()" style="cursor: pointer;">settings</span>
+                    <span class="material-icons float-right ml-0 mt-2" type="button" (click)="openTeamDetails()" style="cursor: pointer;">settings</span>
```
Comment (viv9k) [inline src/app/body/view-organization-details/teams-card/teams-card.component.html:22]: These changes are from other PR

File: src/app/body/board/sprint-details/sprint-details.component.ts [hunk 3]
```diff
@@ -42,6 +42,7 @@ export class SprintDetailsComponent implements OnInit {
   showLoader:boolean = false
   sprintDataReady: boolean = false
   activeSprintNumber: number;
+  today: any = new Date();
```
Comment (viv9k) [inline src/app/body/board/sprint-details/sprint-details.component.ts:45]: There is toolsService you can get the dates from there

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes

**Comment by viv9k** [comment]
change base branch

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required fixed

**Comment by muskan2622** [comment]
required fix is done

**Review by viv9k — APPROVED** [review]
Approved

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**